lib/PINNBench/README.md

```diff
@@ -74,5 +74,18 @@ To install the **modified** `PDEBench` and this benchmark
 python -c "import deephyper_benchmark as dhb; dhb.install('PINNBench/Diffusion-reaction');"
 ```
 
+Run the hyperparameter search
+```
+import os
+os.environ['DEEPHYPER_BENCHMARK_DATASET'] = '2D_diff-react_NA_NA' # set benchmark dataset
+
+import deephyper_benchmark as dhb
+diff_react = dhb.load("PINNBench/Diffusion-reaction")
```

Comment:
looking at the changes it should now be `"PINNBench/Diffusionreaction"` instead of `"PINNBench/Diffusion-reaction"`